content/language/word-list.md

```diff
@@ -83,7 +93,7 @@ In computing, A “master” often refers to the original or main version of an
 * main 
 * original 
 * source
-* control plane
```

Comment:
Same comment as above.

Reply:
This one seems to be more of a typo, but I'll submit it in a separate PR for discussion.